src/Geno/C++/GUI/Widgets/TextEdit.h

```diff
@@ -18,45 +18,194 @@
 #pragma once
 #include <Common/Macros.h>
 #include <Common/Texture2D.h>
-
 #include <filesystem>
+#include <imgui.h>
```

Comment:
Did clang-format put this here, or did you? I usually like to separate STL includes from third-party includes